### ISSUE TITLE
More intuitive image tool behavior

### ISSUE THREAD

**Issue by Munzu** [xournalpp/xournalpp#1963]
**Is your feature request related to a problem? Please describe.**
When I click on the image tool, I expect a window to pop up where I browse for the image to import. I forget that I have to click on the paper for it to pop up.

After I've imported image, I intuitively think my pen is selected again. 
However, the way the image tool works now, I have to **deselect** the image tool before I can write again. This behavior is very counter intuitive. Whenever I forget to deselect the image tool and try to write on the page, the window pops up where I browse for an image to import.
I have never had to import multiple images in a row so I expect the image tool to switch back to the previous tool.

**Describe the solution you'd like**
When clicking on the image tool, a window should pop up to select an image **without** having to click on the paper. After importing, automatically switch back to the previous tool.

**Describe alternatives you've considered**
A feature to import an image by going to `File` > `Import image`

**Comment by Technius** [comment]
> When clicking on the image tool, a window should pop up to select an image without having to click on the paper.

I agree that this is unintuitive. I believe that a click is used so that the image will be inserted at the point of click instead of at some arbitrary place on the canvas. An alternative is to show the popup first and, once the image is selected, show a preview of the image on the canvas.

> I have never had to import multiple images in a row so I expect the image tool to switch back to the previous tool.

In cases where you actually *do* need to import multiple images in a row, the current behavior would be more useful for workflow. Importing a single image is pretty rare, so I think this is an acceptable tradeoff. We could also make this an option in the settings.

> A feature to import an image by going to `File > Import image`

This is a good idea.

**Comment by Munzu** [comment]
Another solution could be to have a dropdown menu for the image tool, just like the pen, eraser, rectangle and selection tools do. When clicking on that downward arrow, one could choose between "Import single image" and another option that would be the current behavior. 
After selecting one of those two options, it would stay that way until changed.
